EIPS/eip-888.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 888
```

Comment:
```suggestion
eip: 4974
```
EIP number is determined by the number assigned to the PR, see [EIP-1](https://eips.ethereum.org/EIPS/eip-1)